hirosystems/bitcoin-core/values.yaml

```diff
@@ -653,8 +653,8 @@ volumePermissions:
   ##
   image:
     registry: docker.io
-    repository: bitnami/bitnami-shell
-    tag: 11-debian-11-r87
+    repository: bitnami/os-shell
+    tag: 11-debian-11-r89
```

Comment:
Any reason why `r89` was chosen? The latest tag is `11-debian-11-r110`, maybe we should update to the latest instead?

Reply:
I'm usually not a big fan of jumping so many version so I wanted to use a version as close as possible to the current version deployed.  If you think it's okay to use the latest and I can push the change.

Reply:
I'm pretty sure it's just security patches from the upstream distro base image, so it should be fine. But am also fine if you'd prefer not to.

Reply:
I just pushed a change to use the latest.